### ISSUE TITLE
Support for stdin/stdout?

### ISSUE THREAD

**Issue by fsouza** [asottile/setup-cfg-fmt#126]
👋 Thanks for setup-cfg-fmt, I'd be curious if you'd accept a patching adding support for reading the file from stdin and writing the output to stdout?

The use-case is for editor integration x)

**Comment by asottile** [comment]
feel free to send a patch similar to what is done here:

- https://github.com/asottile/pyupgrade/blob/55754c9dbd14eade499d524c37272807c25b4d07/pyupgrade/_main.py#L865-L869
- https://github.com/asottile/pyupgrade/blob/55754c9dbd14eade499d524c37272807c25b4d07/pyupgrade/_main.py#L892-L893

**Comment by fsouza** [comment]
Started poking around between meetings and noticed that some fields are filled from adjacent files (readme, license and tox). I think for those cases we have two options:

1. skip them when input is stdin or
2. add some sort of `--stdin-file-name` or something like that to inform what's the file being formatted?

What do you think?

(oh, for the same motivation, I was also thinking of adding `--exit-zero-even-if-changed`, let me know if I should stop lol)

**Comment by asottile** [comment]
>  some fields are filled from adjacent files

hmmmm yeah that would kinda make this intractable wouldn't it :(

**Comment by fsouza** [comment]
> hmmmm yeah that would kinda make this intractable wouldn't it :(

Yeah after seeing that, I'm thinking of leaving it alone (which is option 3 not listed above 😁)
